src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -52,6 +52,7 @@ public CommandResult execute(Model model) {
             return new CommandResult(String.format(LIST_TAGS_MESSAGE_SUCCESS, allTags));
 
         } else {
+            model.updateToDefaultSortedTaskList();
             model.updateFilteredTaskList(PREDICATE_SHOW_ALL_TASKS);
```

Comment:
Do you think we should be using a single level of abstraction here? Maybe it would be good to define a `model#updateToDefaultFilteredTaskList` here!

Reply:
Ah, the suggestion was for line 56, where I thought that it could be abstracted into a function so that both lines follow a single level of abstraction. Apologies, should have made it clearer! 😅 

```suggestion
            model.updateToDefaultSortedTaskList();
            model.updateToDefaultFilteredTaskList();
```

Reply:
Oh, I understand now. Will make the necessary changes.